apps/ingest-from-bucket/README.md

```diff
@@ -116,6 +116,39 @@ docker run \
 
 See [Common Parameters](../../README.md#common-parameters) for additional options.
 
+## Permissions
+
+In S3, the minimal permissions for running the workflow are as follows:
```

Comment:
Should explain where to use this JSON.

Reply:
I'm putting that in the site docs; we also don't explain at all how to get credentials here